src/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -425,7 +425,7 @@ private CompressionMethodValues CompressionMethod
             {
                 if (value == CompressionMethodValues.Deflate)
                     VersionToExtractAtLeast(ZipVersionNeededValues.Deflate);
-                _storedCompressionMethod = value;
+                 _storedCompressionMethod = value;
```

Comment:
Yeah that happens some time. I use the VS comparison tool and it ignores/doesn't highlight whitespace changes like this.